ansible/roles/nginx-rails/tasks/main.yml

```diff
@@ -1,6 +1,8 @@
 ---
 - name: Remove the app's symlink, if exists
-  command: rm -rf /etc/nginx/sites-enabled/{{project_name}}
+  file:
+    path: /etc/nginx/sites-enabled/{{project_name}}
```

Comment:
doesn't this thing happen later in this file? i think we don't want to always remove this anymore